crates/rpc/rpc/src/otterscan.rs

```diff
@@ -0,0 +1,85 @@
+#![allow(dead_code, unused_variables)]
+use async_trait::async_trait;
+use jsonrpsee::core::RpcResult;
+use reth_primitives::{Address, BlockId, TxHash, H256, U256};
+use reth_rpc_api::{EthApiServer, OtterscanServer};
+use reth_rpc_types::{
+    BlockDetails, ContractCreator, InternalOperation, TraceEntry, Transaction,
+    TransactionsWithReceipts,
+};
+
+/// Otterscan Api
+#[derive(Debug)]
+pub struct OtterscanApi<Eth> {
+    eth: Eth,
+}
+
+impl<Eth> OtterscanApi<Eth> {
+    /// Creates a new instance of `Otterscan`.
+    pub fn new(eth: Eth) -> Self {
+        Self { eth }
+    }
+}
+#[async_trait]
+impl<Eth> OtterscanServer for OtterscanApi<Eth>
+where
+    Eth: EthApiServer,
```

Comment:
this might not be enough to fully support the otterscan API, but it is sufficient for now,

but can reevaluate this later